README.md

```diff
@@ -58,6 +58,29 @@ Here's the implementation file. This view's height can change based on the Dynam
 
 If you're following along closely, you may have noticed that there's no reason you _need_ to create a new YOView subclass to use YOLayout. For simple views you can instantiate a YOView and set its layout without creating a new YOView subclass.
 
+## Cocoa
```

Comment:
I might header this section as 'OSX' or 'OSX Support'. I think I hear 'Cocoa' used colloquially to also talk about 'Cocoa Touch' even though it's not technically correct.

Don't feel strongly though, feel free to disregard.